src/content/ja/updates/2019/02/rendering-on-the-web.md

```diff
@@ -1,12 +1,7 @@
-project_path: /web/_project.yaml
-book_path: /web/updates/_book.yaml
+project_path: "/web/_project.yaml"
+book_path: "/web/updates/_book.yaml"
```

Comment:
@ilyaspiridonov these quotes are added but GitLocalize. Can we remove it?

Reply:
@agektmr This is how GitLocalize Markdown module works and it can't be easily removed.

Reply:
Let me try to put the files in to see if this is a semantic error or due to the missing file.

Reply:
@svasilenkov Actually, all I can do is to send a pull request to the original repository's branch `mt-gitlocalize:gitlocalize-9319`. Is it possible to add me to the mt-gitlocalize org so I can easily make changes?